Casks/pharo-launcher.rb

```diff
@@ -1,6 +1,6 @@
 cask 'pharo-launcher' do
-  version '2.0'
-  sha256 '1da8ef2752b359b804072ba692990e989be4c66d6a761a50ab0693082ac805c7'
+  version '2.2'
+  sha256 '99edeafeeaee64958ef14287791df2228c1d5c8ad83732452d27b471c35dac52'
```

Comment:
It looks like you took the last commit of the dev branch.
Pharo Launche 2.2 sha is 41fc4b1665cb08bb912633e1638827f9896b1087

Reply:
@demarey this is not the git commit/tag sha

It is the sha256 from the generated binary bundle available at `https://files.pharo.org/pharo-launcher/2.2/PharoLauncher-2.2-x64.dmg`

Reply:
oh, sorry for the misunderstanding. Then it's fine for me. I checked the sha sum.